Client/Frontend/Sync/SyncSettingsTableViewController.swift

```diff
@@ -258,25 +307,57 @@ class SyncSettingsTableViewController: UITableViewController {
         }
     }
     
-    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+    private func configureToggleCell(_ cell: UITableViewCell, for syncType: SyncDataTypes) {
+        let toggle = UISwitch().then {
+            $0.addTarget(self, action: #selector(didToggleSyncType), for: .valueChanged)
+            $0.isOn = isOn(syncType: syncType)
+            $0.tag = syncType.rawValue
+        }
         
-        switch section {
-        case Sections.deviceList.rawValue:
-            return Strings.devices.uppercased()
-        default:
-            return nil
+        cell.do {
+            $0.textLabel?.text = syncType == .bookmarks ? Strings.bookmarks : Strings.historyMenuItem
+            $0.accessoryView = toggle
+            $0.selectionStyle = .none
+        }
+        
+        cell.separatorInset = .zero
+        
+        func isOn(syncType: SyncDataTypes) -> Bool {
+            switch syncType {
+                case .bookmarks:
+                    return Preferences.Chromium.syncBookmarksEnabled.value
+                case .history:
+                    return Preferences.Chromium.syncHistoryEnabled.value
+            }
+        }
+    }
+    
+    private func configureInformationView(with info: String) -> UITextView {
```

Comment:
Name should probably be like `makeInformationTextView(with info: String)` instead

Reply:
Fixed at https://github.com/brave/brave-ios/pull/3703/commits/86bbfe1e0a6830bff924eebb8f33ee6518107585